StarWars.Core/ViewModels/PlanetsViewModel.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Collections.ObjectModel;
+using System.Threading.Tasks;
+using MvvmCross.Commands;
+using MvvmCross.Navigation;
+using MvvmCross.ViewModels;
+using StarWars.Core.Models;
+using StarWars.Core.Services;
+using StarWars.Core.Services.Abstract;
+
+namespace StarWars.Core.ViewModels;
+
+public sealed class PlanetsViewModel : MvxViewModel
+{
+    private readonly IMvxNavigationService _navigationService;
+    private readonly ISwapiService _swapiService;
+    public IMvxAsyncCommand<PlanetModel> SelectPlanetCommand { get; }
+
+    public PlanetsViewModel(IMvxNavigationService navigationService, ISwapiService swapiService)
+    {
+        _navigationService = navigationService;
+        _swapiService = swapiService;
+        
+        SelectPlanetCommand = new MvxAsyncCommand<PlanetModel>(async planet =>
+        {
+            if (planet != null)
+            {
+                await navigationService.Navigate<PlanetDetailViewModel, PlanetModel>(planet);
+            }
+        });
+    }
+    
+    public ObservableCollection<PlanetModel> Planets { get; } = [];
```

Comment:
You shoulkd not use Models as items in collection for ui binding. 
It violates MVVM architecture.
Here should be PlanetItemViewModel, that keeps only necessary things for UI

Reply:
Also check it on other screens